runtime/avn/src/lib.rs

```diff
@@ -979,6 +979,15 @@ impl_runtime_apis! {
         ) -> Result<(),()>{
             EthBridge::submit_vote(author, events_partition, signature.into())
         }
+
+        fn submit_latest_ethereum_block(
```

Comment:
do you need this to be consumed by the `ethereum_event_handler`

Reply:
you mean the client side?